sql/core/benchmarks/InitCapBenchmark-results.txt

```diff
@@ -0,0 +1,168 @@
+================================================================================================
+[wc=1, wl=1, capitalized=true]
+================================================================================================
+
+OpenJDK 64-Bit Server VM 17.0.11+10-LTS on Linux 5.15.0-122-generic
+Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz
+InitCap evaluation [wc=1, wl=1, capitalized=true]:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+---------------------------------------------------------------------------------------------------------------------------------
+execICU                                                        0              0           0  371177345.1           0.0       1.0X
```

Comment:
Let's bump number of iterations to see seconds in Best/Avg Time.

Reply:
I adjusted the word count for my `Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz`, but encountered issues with local evaluation. This led to a remote evaluation on an `Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz`, where the performance was noticeably less impressive.